Makefile

```diff
@@ -133,7 +130,8 @@ install-arm64: build-arm64
 	sudo install -D -m 755 ./bin/device-worker-aarch64 $(LIBEXECDIR)/yggdrasil/device-worker
 
 uninstall: clean
-	sudo rm -rf $(SYSCONFDIR)/yggdrasil/device/*
+	rm -rf $(SYSCONFDIR)/yggdrasil/device/*
```

Comment:
sudo? it will not be supported as a user without access write to those folders.

Reply:
I mean, we shouldn't use sudo on makefiles at all.To uninstall you can should type `sudo make uninstall` IMHO.

Reply:
in that case we'd be better with getting rid of all the `sudo`, document the need to run specific rules with `sudo` and enforce from within the the makefile having running as root, e.g. by:

```
ifneq ($(shell id -u), 0)
    @echo "You must be root to perform this action."
else
    rm -rf $(SYSCONFDIR)/yggdrasil/device/*
    rm -rf $(LIBEXECDIR)/yggdrasil/device-worker
endif
```

WDYT?